server/utils/azure-appinsights.ts

```diff
@@ -0,0 +1,39 @@
+import appInsights from 'applicationinsights'
+import applicationVersion from '../application-version'
+
+if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY) {
```

Comment:
won't this get run for the job too? meaning application insights starts twice?

Reply:
Isn't the job run in a separate process?  If not then I've made a mistake - or I misunderstand when code is loaded/run - or both!